src/scenes/NftDetailPage/NavigationHandle.tsx

```diff
@@ -53,30 +68,67 @@ const StyledTextWrapper = styled.div<{ direction: Directions }>`
   display: flex;
   margin: auto;
   flex-direction: ${({ direction }) => (direction ? 'row-reverse' : 'row')};
-`;
+  position: absolute;
+  z-index: 100;
+  color: ${colors.gray50};
+  padding: 16px;
+  margin: 0 -16px -16px -16px;
```

Comment:
This adds a larger tap area to the arrow, but does not change its visual position thanks to negative margin